fmn/web/app.py

```diff
@@ -569,14 +569,16 @@ def example_messages(openid, context, filter_id, page, endtime):
         ))
 
     try:
-        total, pages, messages = datanommer.models.Message.grep(
-            start=end-delta,
-            end=end,
-            rows_per_page=bazillion,
-            page=page,
-            order='desc',
-            **hinting
-        )
+        if config.app_conf['datanommer.enabled']:
+            total, pages, messages = datanommer.models.Message.grep(
+                start=end-delta,
+                end=end,
+                rows_per_page=bazillion,
+                page=page,
+                order='desc',
+                **hinting
+            )
+        raise Exception("Datanommer is not enabled.")
```

Comment:
Aren't you missing an `else:` here? Looks like the exception is raised unconditionaly.

Reply:
:man_facepalming: